plugins/spu2-x/src/Windows/Config.cpp

```diff
@@ -294,6 +305,19 @@ BOOL CALLBACK ConfigProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
                     EndDialog(hWnd, 0);
                     break;
 
+                case IDC_OUTPUT:
+                    if (wmEvent == CBN_SELCHANGE) {
+                    const int module = (int)SendMessage(GetDlgItem(hWnd, IDC_OUTPUT), CB_GETCURSEL, 0, 0);
+                    bool IsConfigurable = module != 0;
+                    bool XAudio2Expansion = module == 1 || module == 4;
```

Comment:
Badly named variable since it's not just XAudio2 expansion.

Also, hardcoded values aren't great.